src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/AbstractBuildEventValueProvider.AbstractBuildEventHelper.cs

```diff
@@ -78,7 +79,7 @@ private string GetFromTargets(ProjectRootElement projectXml)
 
                 if (execTask.Parameters.TryGetValue(Command, out string commandText))
                 {
-                    return commandText;
+                    return ProjectCollection.Unescape(commandText);
```

Comment:
I don't think this is correct; if I enter:

```echo %24(Bar)```

This will output:

```$(Bar)```

If we read that value, we'll un-encode the `%24` into `$` which will change the meaning of the line.

I think you literally just want the reverse of the encoding.